sql/core/src/main/scala/org/apache/spark/sql/execution/python/ExtractPythonUDFs.scala

```diff
@@ -113,6 +113,7 @@ object ExtractPythonUDFs extends Rule[SparkPlan] with PredicateHelper {
   def apply(plan: SparkPlan): SparkPlan = plan transformUp {
     // FlatMapGroupsInPandas can be evaluated directly in python worker
     // Therefore we don't need to extract the UDFs
```

Comment:
`FlatMapGroupsInPandas` and `AggregateInPandasExec` can be...

Reply:
Added